gpii/node_modules/flowManager/src/SolutionsRegistryDataSource.js

```diff
@@ -50,15 +50,16 @@ fluid.defaults("gpii.flowManager.solutionsRegistry.dataSource", {
 
 // TODO: Add an invoker to reload once we are "more live".
 gpii.flowManager.solutionsRegistry.dataSource.loadSolutionsRegistry = function (that) {
```

Comment:
Currently, what SolutionsRegistryDataSource.get() does is:
1. If an argument of "os" is given, returns the SR for this platform;
2. If no argument is given, returns a fulllll list of SRs including for windows, linux, darwin, android etc.

I did a search across the code base, point 2 has never been used.

As now only the SR for the platform that Morphic is running on will be fetched from the remote Github, I think it's a good time to refactor this API:
1. SolutionsRegistryDataSource will auto detect the current platform via platform detector;
2. SolutionsRegistryDataSource.get() only returns the SR for the current platform, no argument is needed.